addons/centered/src/components/Centered.svelte

```diff
@@ -0,0 +1,17 @@
+<div class="svelte-centered-wrapper" style="{centeredStyles.style}">
+  <div class="svelte-centered-container" style="{centeredStyles.innerStyle}">
+    <svelte:component this="{OriginalComponent}" {...originalData} />
+  </div>
```

Comment:
Does not work in conjunction with `actions` as events are not forwarded from the inner component.

Reply:
This will be fixed in https://github.com/plumpNation/storybook/pull/1